pallets/phala/src/compute/wrapped_balances.rs

```diff
@@ -192,11 +454,15 @@ pub mod pallet {
 		#[frame_support::transactional]
 		pub fn wrap(origin: OriginFor<T>, amount: BalanceOf<T>) -> DispatchResult {
 			let user = ensure_signed(origin)?;
+			ensure!(
```

Comment:
I think we can allow wrap when there's debt, because in that case, we allow users to deposit more PHA to return the debt.

Reply:
done